docs/how-to-guides/develop-a-flow/referencing-external-files-or-folders-in-a-flow.md

```diff
@@ -0,0 +1,69 @@
+# Referencing external files/folders in a flow
+
+Sometimes, pre-existing code assets are essential for the flow reference. It's common to reference external files or folders that are located outside of the current flow folder. To facilitate this, we allow you to reference these external files or folders using **additional includes** in your flow configuration.
```

Comment:
we might say that creating a python package is the most recommend way. 
And user could do additional includes, if it's too heacy to maintain an python pacakge.

Reply:
Have made an update.